system_test/helpers/aest_docker.erl

```diff
@@ -197,6 +200,21 @@ kill_node(#{container_id := ID, hostname := Name} = NodeState) ->
     log(NodeState, "Container ~p [~s] killed", [Name, ID]),
     NodeState.
 
+node_logs(#{container_id := ID} = _NodeState) ->
+    aest_docker_api:container_logs(ID).
+
+extract_archive(#{container_id := ID, hostname := Name} = NodeState, Path, Archive) ->
+    ok = aest_docker_api:extract_archive(ID, Path, Archive),
+    log(NodeState, "Extracted archive of size ~p in container ~p [~s] at path ~p", [byte_size(Archive), Name, ID, Path]),
+    NodeState.
+
+run_cmd_in_node_dir(#{container_id := ID, hostname := Name} = NodeState, Cmd) ->
+    log(NodeState, "Running command ~p on container ~p [~s]", [Cmd, Name, ID]),
+    Cmd1 = lists:flatten(io_lib:format("docker exec ~s ~s", [ID, lists:join($ , Cmd)])),
```

Comment:
I am adding a command exec using the HTTP API, the idea was to not use the command line API.